src/test/scala/com/high-performance-spark-examples/CustomDataFrameSuiteBase.scala

```diff
@@ -0,0 +1,12 @@
+package com.highperformancespark.examples
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+
+trait CustomDataFrameSuiteBase extends DataFrameSuiteBase {
+
+  override def beforeAll() {
+    super.beforeAll()
+    sc.setLogLevel(org.apache.log4j.Level.WARN.toString)
```

Comment:
O yeah actually I whould probably merge in the code that pr I put out. I
think it fixed that typo.

On Mon, Jan 4, 2016 at 10:10 PM, Holden Karau <notifications@github.com>
wrote:

> In
> src/test/scala/com/high-performance-spark-examples/CustomDataFrameSuiteBase.scala
> <https://github.com/high-performance-spark/high-performance-spark-examples/pull/6#discussion_r48815078>
> :
>
> > @@ -0,0 +1,12 @@
> > +package com.highperformancespark.examples
> > +
> > +import com.holdenkarau.spark.testing.DataFrameSuiteBase
> > +
> > +trait CustomDataFrameSuiteBase extends DataFrameSuiteBase {
> > +
> > +  override def beforeAll() {
> > +    super.beforeAll()
> > +    sc.setLogLevel(org.apache.log4j.Level.WARN.toString)
>
> I think we can do this with a log4j properties file instead of changing
> the base class here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/high-performance-spark/high-performance-spark-examples/pull/6/files#r48815078>
> .
>